### PR TITLE
Cosmos diagnostic fixes

### PR DESCRIPTION
**Pull request by simplynaveen20**
This pr contains three fixes.
 1. https://github.com/Azure/azure-sdk-for-java/issues/13336 Adding time unit in request latency.
 2. https://github.com/Azure/azure-sdk-for-java/issues/13095 Removing duplicate uri string from store result.
 3. Converting diagnostics for point operations in json format (This was the bug introduced when userAgent string added in diagnostics).

closes https://github.com/Azure/azure-sdk-for-java/issues/13336, closes https://github.com/Azure/azure-sdk-for-java/issues/13095

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java [hunk 1]
```diff
@@ -299,6 +303,15 @@ private void validateTransportRequestTimelineDirect(String diagnostics) {
         assertThat(diagnostics).contains("\"eventName\":\"completed\"");
     }
 
+    private void validateJson(String jsonInString) {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:308]: this is test and it doesn't matter much. 
but in general you should not create a ObjectMapper per method invocation, This is costly timewise.

Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java:308]: Yes good catch, although its test, but it should be on class level , i will change it in next iteration

**Pull request merged**